lib/elixir/test/elixir/code_formatter/calls_test.exs

```diff
@@ -414,37 +414,6 @@ defmodule Code.Formatter.CallsTest do
       assert_same "foo(bar)", locals_without_parens: [foo: :*]
     end
 
-    test "without parens on unique argument" do
```

Comment:
We need to rewrite those assertions, they are not specific for check and all. Replace `check` by `assert` and `all` by `for` and I think we should be good.

Reply:
Ah, got it. Updated!